Trivia Maze/src/application/Main.java

```diff
@@ -86,7 +86,6 @@ else if (stage == null)
 		gameMaze = new Maze(ROWS, COLS);
 		player = new Player();
 		database = new Database(type);
-		dataType = type;
```

Comment:
Why remove the dataType assignment? I use that when picking what bg music should be playing.